site/tutorials/snippets/14/quality-gates/setupQualityGate.md

```diff
@@ -131,7 +131,7 @@ Duration: 3:00
 
 Take a look in the Keptn's bridge and navigate to the last deployment. You will find a quality gate evaluation that got a `fail` result when evaluation the SLOs of our carts microservice. Thanks to this quality gate the slow build won't be promoted to production but instead automatically rolled back.
 
-To verify, the [Keptn's Bridge](https://keptn.sh/docs/0.14.x/reference/bridge/) shows the deployment of v0.13.2 and then the failed test in staging including the roll-back.
+To verify, the [Keptn's Bridge](https://keptn.sh/docs/0.14.x/bridge/) shows the deployment of v0.13.2 and then the failed test in staging including the roll-back.
```

Comment:
Just checked, the Keptn 0.17.x entry for the bridge does exist, so maybe we want to use it:
```suggestion
To verify, the [Keptn's Bridge](https://keptn.sh/docs/0.17.x/bridge/) shows the deployment of v0.13.2 and then the failed test in staging including the roll-back.
```

Reply:
This is a change for the tutorial version `0.14` (see file path) since the link did not work and has nothing to do with the new version of the tutorial.

Reply:
Oh true, sorry about that